templates/kubernetes/terraform/modules/kubernetes/main.tf

```diff
@@ -1,11 +1,20 @@
 
-module "monitoring" {
-  source             = "./monitoring"
+module "logging_cloudwatch" {
+  count              = var.logging_type == "cloudwatch" ? 1 : 0
+  source             = "./logging/cloudwatch"
   environment        = var.environment
   region             = var.region
   cluster_name       = var.cluster_name
 }
 
+module "logging_kibana" {
+  count              = var.logging_type == "kibana" ? 1 : 0
+  source             = "./logging/kibana"
+  environment        = var.environment
+  region             = var.region
+  cluster_name       = "${var.project}-${var.environment}-logging"
```

Comment:
would https://github.com/commitdev/zero-aws-eks-stack/blob/526ba45b4fc21b1c08f0a270351b522f99aec4e5/templates/kubernetes/terraform/modules/kubernetes/logging/kibana/variables.tf need to have cluster_name? maybe its meant to be ` elasticsearch_domain`? since it doesn't have a default

Reply:
Yes it is, thanks.